.semaphore/semaphore.yml

```diff
@@ -101,7 +101,7 @@ blocks:
           commands:
             - cd users
             - cache restore linter
-            - './bin/golangci-lint run'
+            - './bin/golangci-lint run
```

Comment:
Is this a typo? What's the reason for this change?

Reply:
Yes it is, let me resolve